src/sagemaker/serve/utils/predictors.py

```diff
@@ -225,18 +227,14 @@ def __init__(
 
     def predict(self, data):
         """Placeholder docstring"""
-        return [
-            self.deserializer.deserialize(
-                io.BytesIO(
-                    self._mode_obj._invoke_multi_model_server_serving(
-                        self.serializer.serialize(data),
-                        self.content_type,
-                        self.deserializer.ACCEPT[0],
-                    )
-                ),
-                self.content_type,
-            )
-        ]
+        logger.info("Entering predict to make a prediction on ")
+        logger.info(data)
```

Comment:
Do we need these logs?

Reply:
They aren't necessary, I will remove them